CONTRIBUTING.rst

```diff
@@ -526,6 +526,41 @@ checkout of MLflow:
 These commands generate a new migration script (e.g., at ``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``)
 that you should then edit to add migration logic.
 
+Docker development (experimental)
```

Comment:
```suggestion
Developing inside a Docker container (experimental)
```

@hubertzub-db What do you think about this section title?

Reply:
Your proposition is certainly more informative.